20.04/ubuntu-debug/Dockerfile

```
@@ -1,11 +1,11 @@
-FROM quay.io/cybozu/golang:1.18-focal AS builder
+FROM --platform=$BUILDPLATFORM quay.io/cybozu/golang:1.18-focal AS builder
 
 WORKDIR /go/src/github.com/fullstorydev/grpcurl
 ARG GRPCURL_VERSION=1.8.7
```

The reference manual says that we need `ARG TARGETARCH` inside of a build stage.
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

```suggestion
ARG GRPCURL_VERSION=1.8.7
ARG TARGETARCH
```

---

Sorry, I added `ARG TARGETARCH` stages to `Dockerfile`.